packages/client-sdk-web/src/internal/pubsub-client.ts

```diff
@@ -217,13 +223,22 @@ export class PubsubClient<
       }
 
       const serviceError = err as unknown as RpcError;
+      const isBrowserDisconnect =
+        serviceError.code === StatusCode.UNKNOWN &&
+        serviceError.message === PubsubClient.BROWSER_DISCONNECT;
       const isRstStreamNoError =
         serviceError.code === StatusCode.INTERNAL &&
         serviceError.message === PubsubClient.RST_STREAM_NO_ERROR_MESSAGE;
+      const shouldReconnectSubscription =
+        isBrowserDisconnect || isRstStreamNoError;
       const momentoError = new TopicSubscribe.Error(
         cacheServiceErrorMapper(serviceError)
       );
-      this.handleSubscribeError(options, momentoError, isRstStreamNoError);
+      this.handleSubscribeError(
+        options,
+        momentoError,
+        shouldReconnectSubscription
+      );
```

Comment:
this shouldn't be showing up in your PR, this is from someone else's. it looks to me like you must have done a merge or something instead of a rebase. we are going to need to fix it